### DISCUSSION TITLE
trying to use "contains" on a dynamic id of a checkbox

### DISCUSSION THREAD

**Discussion by jorisstander** [seleniumbase/SeleniumBase#2486] [Q&A] [answered]
Hi, I've been trying to make seleniumbase press a checkbox that changes it's id every time the page refreshes. The css selector looks like this `#checkbox-14`. The number after - (14) keeps changing and i was wondering if it was possible to use contains here for the id. 
I tried the following but it could not find it this way:
`sb.click('input:contains("checkbox")')`

**Comment by mdmintz** [comment, marked as answer]
That would just be selector of type: `TAG[ATTRIBUTE^="VALUE"]`.

In your case: `input[id^="checkbox"]`
(for an `input` element that has an `id` that starts with `checkbox`)

Your full method call: `sb.click('input[id^="checkbox"]')`

Learn your CSS Selectors: https://www.w3schools.com/cssref/css_selectors.php

**Reply by jorisstander** [reply]
Thanks i see.
